cmd/aws-iam-authenticator/root.go

```diff
@@ -59,6 +60,9 @@ func init() {
 
 	rootCmd.PersistentFlags().StringP("log-format", "l", "text", "Specify log format to use when logging to stderr [text or json]")
 
+	rootCmd.PersistentFlags().String("log-level", "info", "Specify log level to use when logging to stderr ["+strings.Join(getValidLogLevels(), ", ")+"]")
```

Comment:
nit: can we use StringP with shorthand as `v` ?

Reply:
Sure I can add a shorthand, but wouldn't it be clearer to rename the parameter `log-verbosity` or `verbosity`for clarity ?